proxy/src/auth/credentials.rs

```diff
@@ -120,7 +125,17 @@ impl<'a> ClientCredentials<'a> {
 
         info!(user, project = project.as_deref(), "credentials");
 
-        Ok(Self { user, project })
+        let cache_key = format!(
+            "{}{}",
+            project.as_deref().unwrap_or(""),
+            neon_options(params).unwrap_or("".to_string())
```

Comment:
`.unwrap_or_default()`?

Reply:
If you think so, I can change it, Default for strings doesn't add a lot of meaning IMO

Reply:
it's NIT, up to you